package.json

```diff
@@ -24,13 +24,13 @@
   "license": "AGPL-3.0",
   "dependencies": {
     "@babel/plugin-transform-runtime": "7.12.1",
-    "@edx/brand": "npm:@edx/brand-edx.org@^2.0.7",
+    "@edx/brand": "npm:@edx/brand-openedx@1.2.0",
```

Comment:
The Open edX theme should always be installed by default for any MFE in the openedx Github organization. It gets aliased at build time in GoCD with `@edx/brand-edx.org`. These 2 packages share the same interface.